over.css

```diff
@@ -0,0 +1,73 @@
+*{
```

Comment:
In general, our style convention (and most places) is to add spaces between pretty much everything. It may feel icky at first, but you eventually get used to it and it is usually easier to read quickly. In general, there is no objective universal right style; but the right style is the style the codebase is already using (unless it's obviously objectively terrible, like putting multiple statements on one line). So just copy the style that we've used here; I think it's pretty consistent.